Wrappers/Python/cil/framework/utils.py

```diff
@@ -0,0 +1,13 @@
+def find_key(dic, val):
+    """return the key of dictionary dic given the value"""
+    return [k for k, v in dic.items() if v == val][0]
+
+
+def message(cls, msg, *args):
+    msg = "{0}: " + msg
+    for i in range(len(args)):
+        msg += " {%d}" %(i+1)
+    args = list(args)
+    args.insert(0, cls.__name__ )
+
+    return msg.format(*args )
```

Comment:
This is referenced in DataContainer but can be obliterated in favour of Raise and more precise and regular errors

Reply:
I suggest we open an issue regarding this, and not do this within the scope of this PR